src/encoded/schemas/tracking_item.json

```diff
@@ -357,8 +373,135 @@
                                     }
                                 }
                             }
+                        },
+
+                        "file_downloads_by_country" : {
+                            "type" : "array",
+                            "title" : "File Downloads",
```

Comment:
Extremely minor but make sure spacing is even on all of these, your IDE should show these are off.